lib/instructions.dart

```diff
@@ -390,40 +390,40 @@ class Instruction {
     final opcode2 = OverflowOpcode.from(code);
     if (opcode2 == null) return null;
 
-    switch (opcode2) {
+    return switch (opcode2) {
```

Comment:
This doesn't seem any more expressive or terse than the previous form...

Reply:
But the meaning is much more clear. *THIS* switch is the return value for the function. There is no accidental fall-through or anything. I think it reads much better.